h2o-docs/src/product/data-science/gam.rst

```diff
@@ -16,6 +17,166 @@ GAM supports importing and exporting `MOJOs <../save-and-load-model.html#support
 Defining a GAM Model
 ~~~~~~~~~~~~~~~~~~~~
 
+-  `model_id <algo-params/model_id.html>`__: (Optional) Provide a custom name for the model to use as a reference. By default, H2O automatically generates a destination key.
+
+-  `training_frame <algo-params/training_frame.html>`__: (Required) Specify the dataset used to build the model. **NOTE**: In Flow, the training frame is entered automatically if you click the **Build a model** button from the ``Parse`` cell.
```

Comment:
Amin:

Hannah has changed the parameters so that they will be listed in alphabetical order.  Please change the order back to what she originally has.

Reply:
Hannah also has fixed it. Thanks again Hannah!